docs/benchmark_guide.md

```
@@ -0,0 +1,220 @@
+# レポートをブラウザで開く
+make bench-report
+
+# または手動で開く
+open target/criterion/train_type_repository_bench/report/index.html
+```
+
+### コマンドライン出力
+
+ベンチマーク実行後、以下のような出力が表示されます：
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)